### ISSUE TITLE
Is passing plugin ids to the _init_catalog method needed?

### ISSUE THREAD

**Issue by dschristianson** [BASIN-3D/basin3d#207]
In the core.catalog.CatalogBase.initalize method, a list of plugin_ids is passed to the _init_catalog method (which is specific to the type of catalog being used:

https://github.com/BASIN-3D/basin3d/blob/86bcd0dacdd3c7578fbbbecc7e293f0caff662b4/basin3d/core/catalog.py#L78

In the CatalogSqlAlchemy._init_catalog method, I cannot see where the list of plugin_ids passed in is used.

Also, the CatalogDjango (in django-basin3d) also does not currently make use of such a list.

Thus, does the parameter actually need to be passed in?

**Comment by vchendrix** [comment]
@dschristianson it does not need to be passed in.

**Comment by dschristianson** [comment]
OK, thanks. My sense is that we change in a future release.

**Comment by vchendrix** [comment]
> OK, thanks. My sense is that we change in a future release.

agreed.  It may give us more flexibility in the long run with catalog configuration
